packages/multi-select-combo-box/theme/lumo/vaadin-multi-select-combo-box-styles.js

```diff
@@ -54,6 +54,14 @@ registerStyles(
     :host([auto-expand-vertically]) {
       padding-block: var(--lumo-space-xs);
     }
+
+    #wrapper {
+      max-width: calc(100% - var(--lumo-icon-size-m) - 2 * var(--lumo-space-xs));
```

Comment:
What is this formula based on? Specifically, why does it subtract two `var(--lumo-space-xs)`?

The side paddings of the component appear to be 6px while the gap between the chips and buttons is currently 8px = `2 * var(--lumo-space-xs)`. Shouldn't they match?

<img width="322" alt="image" src="https://github.com/vaadin/web-components/assets/5039436/93901823-c20f-4f46-b3ce-bf00b25a6edc">

Reply:
Removed these calculations in favor of the solution proposed at https://github.com/vaadin/web-components/pull/6905#discussion_r1413676843